### PR TITLE
[ADD] Issue templates for bug report and feature request

### PR DESCRIPTION
**Pull request by joao-p-marques**
As suggested in https://github.com/OCA/maintainer-quality-tools/issues/692

Usage example (similar): https://github.com/Tecnativa/doodba/issues/new/choose

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
Excellent starting point. Thanks!

**Review by sbidoul — APPROVED** [review]
Shouldn't we explicitly ask about the OCA addon and Odoo series ?

Otherwise :+1:

**Comment by joao-p-marques** [comment]
> Shouldn't we explicitly ask about the OCA addon and Odoo series ?

Thanks! Do you mean the Odoo and module version? We do ask for the affected versions [here](https://github.com/OCA/.github/pull/1/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R16), although it could be more explicit, indeed...

We could also consider moving to [Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) instead, although I am not sure these can be set at an organization level like this templates... 

What do you prefer?

**Review by yajo — APPROVED** [review]
We should merge now and fix issues later. This is an improvement already!

**Review by tafaRU — APPROVED** [review]

**Comment by joao-p-marques** [comment]
Cool! Can we merge this, then? I will make the 2 PRs that @sbidoul asked in the [OCA Days chat](https://matrix.to/#/!gKuRXYxJkWkxMhIPXn:matrix.org/$_YviYr1JOXTVdoizyqEprGk3EQJexbG-qcCrEp1XoAU?via=recallstack.icu&via=t2bot.io&via=matrix.org) next week, but in the meantime this can be merged.

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (sbidoul) [inline .github/ISSUE_TEMPLATE/config.yaml]: what does this do ?

Comment (joao-p-marques) [reply .github/ISSUE_TEMPLATE/config.yaml]: This forces the user to choose one of the given options for the issue type and template (Bug Report or Feature Request). If set to `true`, the user would still be able to bypass that and open a "normal" issue: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Comment (sbidoul) [reply .github/ISSUE_TEMPLATE/config.yaml]: Should we not start with a softer approach ? And then disable it later when the issue template have proven to work well ?

Comment (joao-p-marques) [reply .github/ISSUE_TEMPLATE/config.yaml]: Ok, I changed it back to `true`.

**Comment by joao-p-marques** [comment]
@sbidoul I also added here a new PR template, where we can refer to the OCA contribution guidelines, instead of a per-project `CONTRIBUTING.md` file (removed in https://github.com/OCA/oca-addons-repo-template/pull/88)

**Comment by sbidoul** [comment]
Great! I think [CONTRIBUTING.md is useful](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) too, however.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name: Bug report
+about: Report a bug
+title: ""
+labels: bug
+assignees: ""
+---
+
+<!-- Provide a general summary of the issue in the Title above.
+Before opening the issue, please check that there isn't another issue addressing the same problem already created -->
+
```
Comment (sbidoul) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
<!-- Provide a general summary of the issue in the Title above.
Before opening the issue, please check that there isn't another issue addressing the same problem already created -->

## Module
The name of the module that has a bug.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pull Request
+title: ""
+assignees: ""
+---
+
+<!-- Please Please follow the official guide from the OCA Guidelines page (https://odoo-community.org/page/contributing)  -->
```
Comment (sbidoul) [inline .github/pull_request_template.md]: I suggest adding something link "please provide a summary of what your PR does and why it is necessary or useful."

**Comment by joao-p-marques** [comment]
Oh ok. In that case, I added it here, and removed the PR template. I was trying to replace one with the other, but for OCA's use case it makes more sense to keep `CONTRIBUTING.md` only, indeed. The PR templates come handy when you want to be more specific when opening them but, as said, let's not introduce too much friction in the process at once.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# OCA Guidelines
+
+Please follow the official guide from the
+[OCA Guidelines page](https://odoo-community.org/page/contributing).
+
+## Project Specific Guidelines
+
+<!-- /!\ do not modify above this line -->
+
+This project does not have specific coding guidelines.
```
Comment (sbidoul) [inline CONTRIBUTING.md]: Since this is now an org-level file, this section about project-specific guidelines does not make sense anymore.

**Comment by joao-p-marques** [comment]
@sbidoul changes made. Thanks!

**Comment by sbidoul** [comment]
LGTM, let's try this.

**Comment by sbidoul** [comment]
@joao-p-marques do you want to communicate about this on the contributors mailing list ?

**Comment by joao-p-marques** [comment]
> @joao-p-marques do you want to communicate about this on the contributors mailing list ?

Sure, I'll prepare the e-mail :+1:

**Pull request merged**